src/main/kotlin/org/rust/lang/core/psi/ext/RsVisibility.kt

```diff
@@ -33,6 +33,8 @@ fun RsVisibilityOwner.iconWithVisibility(flags: Int, icon: Icon): Icon =
         icon.addVisibilityIcon(isPublic)
 
 fun RsVisible.isVisibleFrom(mod: RsMod): Boolean {
+    if (this is RsFile && declarations.size > 1) return true
```

Comment:
Let's add a TODO comment to fix it because it produces false negative for some cases like
```rust
//- main.rs
use crate::aa::a::x; // <- no error here although compiler produces it

mod aa {
    mod a;
}

fn main() {}

//- lib.rs
use crate::aa::a::x; // <- no error here although compiler produces it

mod aa {
    mod a;
}

//- aa/a.rs
pub fn x() {}
```

Reply:
BTW let's add code above as test